Assignment/solution_student/css/style.css

```diff
@@ -1,27 +1,193 @@
+}
+/* Micro clearfix hack: http://nicolasgallagher.com/micro-clearfix-hack/ */
+.clearfix {
+  *zoom: 1;
+}
+.clearfix:before,
+.clearfix:after {
+    content: " ";
+    display: table;
+}
+.clearfix:after {
+  clear: both;
+}
+
+a:hover {
+    text-decoration: none;
+}
+
+header { 
+ 	background-image: url("../images/header_bg.jpg");
+	width: 100%;
+	height: 500px;
+	background-size: contain;
+	background-repeat: no-repeat;
+	margin: 0;
+  	padding: 0;
+}
+
+h1 {
+	color: white;
+	font-weight: bolder;
+	font-size: 24px;
+	padding-left: 20px;
+	padding-top: 5px;
+}
+
+h2 {
+	color: white;
+	font-weight: semibold;
+	font-size: 14px;
+	border-radius: 5px;
+	text-align: center;
+	padding-top: 100px;
+}
+
+h3 {
+	color: #033048;
+	font-size: 20px;
+	font-weight: semibold;
+	padding: 0;
+	margin: 0;
+}
+
+.container {
+	padding: 0;
+	padding-bottom: 20px;
+	margin: 0;
+	width: 50%;
+	text-indent: 20px;
+}
+
+li {
+	text-indent: 20px;
+}
+
+.nowbutton {
+	text-align: center;
+}
+
+.now{
+	position: relative;
+	top: 75%;
+	padding: 10px 15px 11px;
+	font-size: 14px;
+	background-color: #f9e42e;
+	font-weight: bold;
+	color: #033048;
+	border-radius: 5px;
+	text-decoration: none;
+}
+
 body {
-  color: black;
-  font-size: 16px;
+  font-size: 13px;
+  font-weight: lighter;
   background: white;
+  padding: 50px;
+  color: #05080C;
+  line-height: 1.5;
+  max-width: 900px;
+  margin: 70px auto;
+  background: #F6F6F6;
+  border-top: 5px solid #222222;
+  font-family: 'open sans',sans-serif;
+  border-bottom: 5px solid #222222;
+  padding-top: none;
 }
 
-/******************************************
-/* LAYOUT
-/*******************************************/
 
-/******************************************
-/* PAGES
-/*******************************************/
+.quote {
+	background-image: url("../images/testimonial_bg.jpg");
+  	background-repeat: no-repeat;
+  	background-position: center;
+  	width: 100%;
+  	background-repeat: no-repeat;
+	text-align: center;
+	color: white;
+	padding: 40px;
+}
+
+blockquote {
+  	margin: 10 25% 0 25%;
+  	font-weight: 500;
+  	margin top: 20px;
+  	margin bottom: 0;
+  	display: inline-block;
+}
+
+h4 { 
+	font-weight: bolder;
+	font-size: 18px;
+}
+
+.sign {
+	position: relative;
+	top: 70%;
+	padding: 10px 15px 11px;
+	font-size: 12px;
+	background-color: #033048;;
+	font-weight: bold;
+	color: white;
+	border-radius: 5px;
+	text-decoration: none;
+	display: inline-block;	
+	border: none;
+}
+
+.signupnow {
+	background-color: #f9e42e;
+	background-position: center;
+	text-align: center;
+	height: 70px;
+	margin: 0;
+	padding: 5px 10px 5px 10px;
+	border: none;
+}
+
+.social {
+	text-align: center;
+}
+
+a { 
+	padding-right: 10px;
+	padding-left: 10px;
+}
+
+footer{ 
+	background-color: #708090;
+	margin: 0;
+	border-top: none;
+	border-bottom: none;
+	padding: 0;
+	background-size: 500px;
+	height: 150px;
+}
+
+h5{
+	color: #f9e42e;
+	text-align: center;
+	font-size: 24px;
+	font-weight: extra bold;
```

Comment:
So the extra spacing between the yellow `signupnow` div and the `footer` is being caused by the default top-margin on this `h5`, which is hard to tell, but if you look at the inspector you can see it: [http://d.pr/i/UnM0](http://d.pr/i/UnM0)

Also, `extra bold` isn't a valid value for `font-weight`, so that's why it's not working. Looks like `bold` will have to do here, unless you include a custom font with more variations on the font-weight.